src/Yesod/Markdown.hs

```diff
@@ -50,7 +50,6 @@ import Text.Blaze.Html (preEscapedToMarkup)
 import Text.HTML.SanitizeXSS (sanitizeBalance)
 import Text.Hamlet (hamlet, Html)
 import Text.Pandoc
-import Text.Pandoc.Error
```

Comment:
Does this removal mean we have to bump our _lower_ bounds? I.e. was it needed at earlier versions then became a re-export later?